File: Backend/src/main/java/com/projekat/XML/dtos/CommentDTO.java
package com.projekat.XML.dtos;

public class CommentDTO {

    private String message;

    private Double grade;

    private Long ad;

    private Long userId;

    public CommentDTO(String message, Double grade, Long ad, Long userId) {
        this.message = message;
        this.grade = grade;
        this.ad = ad;
        this.userId = userId;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public Double getGrade() {
        return grade;
    }

    public void setGrade(Double grade) {
        this.grade = grade;
    }

    public Long getAd() {
        return ad;
    }

    public void setAd(Long ad) {
        this.ad = ad;
    }

    public Long getUserId() {
        return userId;
    }

    public void setUserId(Long userId) {
        this.userId = userId;
    }
}

File: Frontend/src/app/dtos/comment-dto.ts
export class CommentDto {
  message: string;
  grade: number;
  ad: number;
  userId: number;

  constructor(message: string, rating: number, ad: number, userId: number) {
    this.message = message;
    this.grade = rating;
    this.ad = ad;
    this.userId = userId;
  }
}
